Smart-Contract-Verifier-Server/src/main/kotlin/com/savannasolutions/SmartContractVerifierServer/negotiation/requests/CreateAgreementRequest.kt

```diff
@@ -2,8 +2,7 @@ package com.savannasolutions.SmartContractVerifierServer.negotiation.requests
 
 import com.fasterxml.jackson.annotation.JsonProperty
 
-data class CreateAgreementRequest(@JsonProperty("PartyA") val PartyA:String,
-                                  @JsonProperty("PartyB") val PartyB:String,
+data class CreateAgreementRequest(@JsonProperty("PartyB") val PartyB:String,
```

Comment:
Is PartyA assumed to be the logged in user?

Reply:
Yes it is. Or well the user sending the request